sklearn/feature_extraction/text.py

```diff
@@ -1377,6 +1436,25 @@ class TfidfVectorizer(CountVectorizer):
 
         This is only available if no vocabulary was given.
 
+    Examples
+    --------
+    >>> from sklearn.feature_extraction.text import CountVectorizer
+    >>> corpus = [
+    ...     'This is the first document.',
+    ...     'This is the second second document.',
+    ...     'And the third one.',
+    ...     'Is this the first document?',
+    ... ]
+    >>> vectorizer = CountVectorizer()
```

Comment:
I would expect to have a TfidfVectorizer in this docstring, isn't it?

Reply:
Same as above: [commit](https://github.com/scikit-learn/scikit-learn/pull/11534/commits/f7fd421f3454aaee9fe4d8f88e8258e36b8b7ca3)